autoload/neomake/makers/ft/scss.vim

```diff
@@ -0,0 +1,13 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#scss#EnabledMakers()
+    return ['scsslint']
+endfunction
+
+function! neomake#makers#ft#scss#scsslint()
+    return {
+        \ 'exe': 'scss-lint',
+        \ 'errorformat': '%f:%l [%t] %m',
+        \ 'returns': [0, 1, 2, 64, 66, 70, 78]
```

Comment:
I'm guessing this was copied from syntastic, but Neomake doesn't use this attribute.